service/build.gradle

```diff
@@ -318,6 +318,8 @@ dependencies {
 
   implementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
 
+  implementation group: 'net.minidev', name:'json-smart', version: '2.3.1'
```

Comment:
the alternative if you don't think it is worth adding the dep here is to add the CVE to `config/owasp/suppressions.xml`